src/pretalx/orga/forms/submission.py

```diff
@@ -150,6 +150,22 @@ def clean(self):
             )
         return data
 
+    def clean_room(self):
```

Comment:
Please move these checks to the `clean()` method, as the `clean_*` methods are not guaranteed to have access to all data yet. Validation that impacts multiple fields is [best done in `clean()`](https://docs.djangoproject.com/en/dev/ref/forms/validation/#cleaning-and-validating-fields-that-depend-on-each-other).